### DISCUSSION TITLE
Tauri mode

### DISCUSSION THREAD

**Discussion by fprijate** [quasarframework/quasar#14999] [Ideas / Proposals]
It would be nice to have tauri mode in Quasar (quasar dev -m tauri)
 Here is my simple template [Quasar Tauri template](https://github.com/fprijate/quasar-tauri-template).
It's just a proof of concept.
It should be integrated , like electron is into quasar.config.js

**Comment by Edge-coordinates** [comment]
> It would be nice to have tauri mode in Quasar (quasar dev -m tauri) Here is my simple template [Quasar Tauri template](https://github.com/fprijate/quasar-tauri-template). It's just a proof of concept. It should be integrated , like electron is into quasar.config.js

So is there an official plan? Would you like to submit an issue?
